etc/kayobe/environments/ci-multinode/kolla.yml

```diff
@@ -2,3 +2,5 @@
 kolla_enable_cinder: true
 kolla_enable_cinder_backup: true
 kolla_enable_neutron_provider_networks: true
+kolla_enable_ovn: true
```

Comment:
Unsure whether we should change this default. Ideally it should be possible & easy to use either OVS or OVN.